logging.js

```diff
@@ -2,7 +2,7 @@
 // https://stackoverflow.com/questions/9781218/how-to-change-node-jss-console-font-color
 
```

Comment:
fixed error in the logging. now it display yellos properly:
![image](https://user-images.githubusercontent.com/97619732/162961827-f49a46f3-4146-4f13-9320-84b3bf6a5948.png)